harness/tests/experiment/integrations/test_deepspeed_trial.py

```diff
@@ -458,6 +459,7 @@ def make_workloads() -> workload.Stream:
         )
         controller.run()
 
+    @pytest.mark.skipif(shutil.disk_usage("/dev/shm")[0] < 10**8, reason="insufficient shm size")
```

Comment:
Could you please drop a TODO in here referencing the ticket to re-enable the test? I've seen (and been guilty of) enough instances where tests are never re-enabled that I think the TODO is a good practice.

Reply:
added